inc/class-customizer-guided-tour.php

```diff
@@ -0,0 +1,213 @@
+<?php
+/**
+ * Customizer Guided Tour Class
+ *
+ * @author   WooThemes
+ * @since    0.8.7
+ */
+
+if ( ! defined( 'ABSPATH' ) ) {
+	exit;
+}
+
+if ( ! class_exists( 'Customizer_NUX_Guided_Tour' ) ) :
+
+	class Customizer_NUX_Guided_Tour {
+		/**
+		 * Setup class.
+		 *
+		 * @since 0.8.7
+		 */
+		public function __construct() {
+			add_action( 'admin_init', array( $this, 'customizer' ) );
+		}
+
+		/**
+		 * Customizer.
+		 *
+		 * @since 0.8.7
+		 */
+		public function customizer() {
+			global $pagenow;
+
+			if ( 'customize.php' === $pagenow && isset( $_GET['wc-api-dev-tutorial'] ) ) {
+				add_action( 'customize_controls_enqueue_scripts',      array( $this, 'customize_scripts' ) );
+				add_action( 'customize_controls_print_footer_scripts', array( $this, 'print_templates' ) );
+				$this->isStorefrontActivated();
+			}
+		}
+
+		public function isStorefrontActivated() {
+			$current_theme = wp_get_theme();
+
+			if ( 'Storefront' == $current_theme->get( 'Name' ) ) {
+				$redirect_url = add_query_arg( array(
+					'sf_guided_tour' => '1',
+					'sf_tasks' => 'homepage',
+					'theme' => $_GET['theme'],
+				), admin_url( '/customize.php' ) );
+
+				wp_redirect( $redirect_url );
+				exit;
+			}
+		}
+
+		/**
+		 * Customizer enqueues.
+		 *
+		 * @since 0.8.7
+		 */
+		public function customize_scripts() {
+			global $storefront_version;
+
+			$suffix = ( defined( 'SCRIPT_DEBUG' ) && SCRIPT_DEBUG ) ? '' : '.min';
+
+			wp_enqueue_style( 'wc-api-dev-guided-tour', WC_API_Dev::$plugin_url . 'assets/css/admin/customizer.css', array(), $storefront_version, 'all' );
+
+			wp_enqueue_script( 'wc-api-dev-guided-tour', WC_API_Dev::$plugin_url . 'assets/js/admin/customizer' . $suffix . '.js', array( 'jquery', 'wp-backbone' ), $storefront_version, true );
```

Comment:
When I uploaded this to my AT site, it didn't work because `customizer.min.js` doesn't exist. I don't think we have any package builder to generate the .min files.